conf/openshift/openshift.json

```diff
@@ -12,6 +12,59 @@
     }
   },
   "objects": [
+    {
+      "kind": "Secret",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "dataverse-admin-secret"
+      },
+      "stringData" : {
+        "admin-password" : "${ADMIN_PASSWORD}"
+      }
+    },
+  	{
+      "kind": "Secret",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "dataverse-postgres-secret"
+      },
+      "stringData" : {
+        "postgres-user" : "${POSTGRES_USER}",
```

Comment:
My initial thinking was that database user names might not be sensitive enough to be treated as secret; but considering it's password auth vs pub key auth (and low-implementation complexity, and already done) it seems reasonable to go with it as is.